README.md

```diff
@@ -289,20 +292,38 @@ Below you can find the list of methods that can be used as expectations:
 
 > Note: if you use such methods as expectations they will not count as expectations, in a test report, for example, but if they fail a clear message will be shown to ease understanding why the test has failed. In the end, test reports are usually useful in cases where tests have failed and we need to understand why. If all tests are green there is nothing to worry about, at least not if they were well implemented.
 
-### Example of a test failure when using such methods as expectations
+### Examples of a test failure when using such methods as expectations
 
 Let's look how a failure would looks like when using some of this methods as expectations.
 
-- Failure when using `waitForElementPresence` as expectation:
+- Failures when using `waitForElementPresence` as expectation:
 
 ```sh
-Failed: element with locator '*[id="foo"]' is not present
+Failed: element with locator 'by.cssContainingText("foo", "bar")' is not present
```

Comment:
This is wrong, isn’t it? As it was in the previous PR.

Reply:
We should make it clear to the user that one message will be shown when locating elements using `by.id`, `by.className`, or `by.css`, and that the other message will be shown when locating elements using `by.cssContainingText`, `by.buttonText`, and `by.partialButtonText`.
Maybe we could find some time to finish this pull request in pair. What do you think? We could try an online pair programming session.

Reply:
I agree with your suggestion. I had some limitations to do the examples because of my level in english.

Reply:
Hm, maybe you should split this in more than one PR then. It's been confusing reviewing it because some parts are documentation related, and others are because of new stuff, such as prettier.